querybook/webapp/components/QueryRunButton/QueryRunButton.tsx

```diff
@@ -204,6 +208,13 @@ const QueryLimitSelector: React.FC<{
         }
     }, [rowLimit, setRowLimit]);
 
+    const selectedRowLimitText = React.useMemo(() => {
+        if (rowLimit != null && rowLimit >= 0) {
```

Comment:
as the `rowLimit` has type of `number`, why need to check if it's null? can't it be simply 
`rowLimit === -1 ? 'none' : formatNumber(rowLimit)`?